File: C++/Daily Flash/Week2/Day1_Jan20/Code/program4.cpp
/*
 * Program 4 : Write a Progr am to accept 10 integers from user and prin t s the
sum & average of entered numbers .
Input: 1 2 3 4 5 6 7 8 9 10
Output:
Sum of 10 entered Number is: 55
Average of 10 entered number is: 5.55
*
*/

#include <iostream>

int main() {
	int sum= 0, num;
	std::cout << "Enter 10 numbers: " << std::endl;

	for(int i=1; i<=10; i++) {
		std::cin >> num;
		sum +=num;
	}
	float avg = sum/10;
	int fraction = sum%10;

	std::cout << "Sum of 10 numbers : " << sum << std::endl;
	std::cout << "Average of 10 numbers : " << avg <<"."<< fraction << std::endl;
	return 0;

}

File: JAVA/Daily Flash Codes/Week2/Day1_Jan20/code/program4.java
/*
*
* Program 4 : Write a Progr am to accept 10 integers from user and prin t s the
sum & average of entered numbers .
Input: 1 2 3 4 5 6 7 8 9 10
Output:
Sum of 10 entered Number is: 55
Average of 10 entered number is: 5.55
*/

import java.io.*;

class Program {
	public static void main(String[] args) throws IOException {
		BufferedReader in = new BufferedReader(new InputStreamReader(System.in));		//obj of Bufferedreader to take input
		int num , sum = 0;
		
		System.out.println("Enter 10 integers: ");
		for(int i=1; i<=10; i++) {
			num = Integer.parseInt(in.readLine());
			sum +=num;
		}
		int avg = sum/10;
		int fraction = sum%10;
		System.out.println(" Sum of 10 numbers: "+sum);
		System.out.println("Average of 10 numbers: "+avg + "." + fraction);

	}


}
